examples/quickstart/notebook.ipynb

```diff
@@ -0,0 +1,307 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Penguins dashboard\n",
+    "\n",
+    "> This dashboard helps you explore the penguins dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from urllib.request import urlretrieve\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "import matplotlib.pyplot as plt\n",
+    "from jinja2 import Template\n",
+    "\n",
+    "from sql.ggplot import ggplot, aes, geom_boxplot, geom_histogram, facet_wrap"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = urlretrieve(\"https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv\", \"penguins.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext sql\n",
```

Comment:
Tried this cell in colab and it throws error:
```
MetaData.__init__() got an unexpected keyword argument 'bind'
Connection info needed in SQLAlchemy format, example:
               postgresql://username:password@hostname/dbname
               or an existing connection: dict_keys([])
```

I run this : `%pip install ipywidgets jupysql duckdb duckdb-engine matplotlib --quiet` before this cell.